cli/container_test.go

```diff
@@ -329,3 +329,97 @@ func Test_parseDeviceMappings(t *testing.T) {
 		})
```

Comment:
@allencloud As you can see, in the unit test case generated by vscode, we can only detect the existence of an error, true or false.

Reply:
yeah, I think it has some drawbacks.